### ISSUE TITLE
Fix warnings for clojure 1.11

### ISSUE THREAD

**Issue by ieugen** [whamtet/ctmx#5]
Hi,

Started a new project and I get these warnings:
```
WARNING: parse-long already refers to: #'clojure.core/parse-long in namespace: ctmx.rt, being replaced by: #'ctmx.rt/parse-long
WARNING: parse-double already refers to: #'clojure.core/parse-double in namespace: ctmx.rt, being replaced by: #'ctmx.rt/parse-double
WARNING: parse-boolean already refers to: #'clojure.core/parse-boolean in namespace: ctmx.rt, being replaced by: #'ctmx.rt/parse-boolean
```
I think they should be fixed eventually.
Having this to track that down.

**Comment by whamtet** [comment]
Fixed in 1.4.5

**Comment by whamtet** [comment]
Thanks for the bug report Eugen. Do you mind sharing a repo so I can reproduce?
